osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -3,26 +3,45 @@
 
 using System;
 using System.Linq;
+using JetBrains.Annotations;
 using osu.Framework.Allocation;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Animations;
 using osu.Framework.Graphics.Containers;
-using osu.Framework.Graphics.Sprites;
 using osu.Framework.Input.Bindings;
 using osu.Framework.Utils;
 using osu.Game.Beatmaps;
 using osu.Game.Rulesets.Catch.Objects;
 using osu.Game.Rulesets.Catch.Objects.Drawables;
+using osu.Game.Rulesets.Catch.Skinning;
 using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Skinning;
 using osuTK;
 using osuTK.Graphics;
 
 namespace osu.Game.Rulesets.Catch.UI
 {
-    public class Catcher : Container, IKeyBindingHandler<CatchAction>
+    public class Catcher : SkinReloadableDrawable, IKeyBindingHandler<CatchAction>
     {
+        /// <summary>
+        /// The default colour used for all hyper-dashing components. (catcher drawables and fruit)
+        /// </summary>
         public static readonly Color4 DEFAULT_HYPER_DASH_COLOUR = Color4.Red;
 
+        /// <summary>
+        /// The default hyper-dash colour used directly for this
+        /// <see cref="Catcher"/>'s <see cref="Drawable.Colour"/>.
+        /// </summary>
```

Comment:
I still think this xmldoc is misleading (if not unreadable - I still don't quite understand what it's trying to say), and more importantly, it directly contradicts the xmldoc of `DEFAULT_HYPER_DASH_COLOUR` immediately preceding it. I've mentioned this once and I'm pretty sure I unresolved the relevant conversation, but the conversation is resolved again and I see no actual changes to the content of the xmldoc, save for insertion of one new line.

Please don't resolve conversations without addressing them in some shape or form.

Reply:
```suggestion
        /// <summary>
        /// The default color used for the direct <see cref="Drawable.Colour"/> of the <see cref="Catcher"/> while in hyper-dashing state.
        /// </summary>
        /// <remarks>
        /// This colour is used specially in here on the default skin only.
        /// </remarks>
```

What do you think?

Reply:
No, I still don't understand what that xmldoc is trying to say. What is a "*direct colour*"? You're still not describing what the colours are actually *colouring*.

If I had to have a go at documenting these constants somewhat accurately (from what I can tell from a quick look), I'd probably write something along the lines of

```csharp
/// <summary>
/// The default colour used to tint hyper-dash fruit, the catcher's trail and its after-image during a hyper-dash.
/// </summary>
public static readonly Color4 DEFAULT_HYPER_DASH_COLOUR = Color4.Red;

/// <summary>
/// The default colour used to tint the moving catcher sprite, along with the fruits on the catcher plate, during a hyper-dash.
/// </summary>
public static readonly Color4 DEFAULT_CATCHER_HYPER_DASH_COLOUR = Color4.OrangeRed;
```

which to me shows quite well how arbitrary and weird this distinction has become (unless I'm wrong about this description, but then it shows that this colour rigamarole is way too complicated).